src/main.rs

```diff
@@ -1,33 +1,71 @@
-fn input() -> String {
-   let mut buf = String::new();
-   std::io::stdin().read_line(&mut buf).unwrap();
-   buf.trim().to_string()
+use std::io::{Stdin, Stdout, Write};
+
+#[derive(Debug, Clone)]
+struct Todo {
+    message: String,
+}
+
+impl Todo {
+    fn new(message: String) -> Todo {
+        return Todo { message };
+    }
 }
 
 fn main() {
     println!("Seja bem vindo ao gerador de TODO 🚀 ");
-    loop {        
-        if todo_start() {            
+    loop {
+        let mut start_terminal = Terminal::new();
```

Comment:
```suggestion
    let mut start_terminal = Terminal::new();
    loop {
```
Essa variável pode ser movida para fora do `loop`, já que só é necessário instanciar o `Terminal` uma vez.